src/Interface/HasRawPartialOrder.agda

```diff
@@ -0,0 +1,15 @@
+{-# OPTIONS --safe #-}
+
+module Interface.HasRawPartialOrder where
+
+open import Level                       using (_⊔_ ; suc)
+open import Relation.Binary             using (Rel)
+
+-- A "raw" partial order is just a relation that we expect to play the role of
+-- a partial order, but does not come with a proof that the relation satisfies
+-- any of the three properties required to be a partial order.
+record HasRawPartialOrder {a ℓ} (A : Set a) : Set (a ⊔ suc ℓ) where
```

Comment:
I think we should do at least one of the following two things:
- rename this record to `HasLeq` (or something similar)
- make a lawful typeclass using `IsPartialOrder` from `Relation.Binary`

The issue with having a raw partial order is that you're promising something that you could also just prove, which I think we should avoid.

Reply:
@WhatisRT, note that by eliminating the "raw partial order" type class we also eliminate the instance of `HasRawPartialOrder TokenAlgebra _≡_`, and we can't make such an instance of the new type class that requires proof of order properties, e.g., `HasPartialOrder TokenAlgebra _≡_`, since `_≤ᵗ_` is abstract.